next.config.js

```
@@ -1,45 +1,56 @@
+    async headers() {
+      return [
+        {
+          source: '/(.*)',
+          headers: [
+            { key: 'X-Frame-Options', value: 'DENY' },
+            { key: 'Content-Security-Policy', value: "frame-ancestors 'none'" },
+          ],
+        },
+      ]
+    },
+    async rewrites() {
```

ESLint

[error] Missing space before function parentheses.

(space-before-function-paren)